transactions.md

```diff
@@ -151,7 +151,7 @@ The transaction which spends the [`vault_tx`](vault_tx) output to the EDV by the
 - count: 1
 - inputs[0]:
     - txid: `<deposit_tx txid>`
-    - vout: N/A
+    - vout: `<deposit_tx outpoint>`
```

Comment:
I thought the [outpoint](https://btcinformation.org/en/glossary/outpoint) was the `TXID` and the `vout`? Let's either combine the `txid` and `vout` fields into an `outpoint` field, or keep them separate (to be consistent with the other transactions for which the `vout` are specified).

Reply:
Yep, typo here good catch :)

Reply:
Went for keeping the separation